deploy.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+while getopts ":e:r:" flag
+do
+    case "${flag}" in
+        e) env=${OPTARG};;
+        r) region=${OPTARG};;
+        *) echo "Invalid option. Only -e and -r are allowed" >&2
+            exit 1 ;;
+    esac
+done
+env=${env:-prod}
+region=${region:-us-east-1}
+
+echo "ENV: $env";
+echo "Region: $region";
+
+# Download model artifacts
+dvc pull
```

Comment:
Mostly the same script as the other repo, but now we can use DVC to pull model artifacts